AK/StringView.cpp

```diff
@@ -63,7 +63,7 @@ Vector<StringView> StringView::split_view(StringView separator, SplitBehavior sp
 {
     Vector<StringView> parts;
     for_each_split_view(separator, split_behavior, [&](StringView view) {
-        parts.append(view);
+        parts.try_append(view).release_value_but_fixme_should_propagate_errors();
```

Comment:
I'm not really sure if this is the right solution. Maybe we should instead not allow `split_view()` in the kernel at all. Directly using `for_each_split_view()` should be enough for most use cases in the kernel, otherwise we could maybe add `try_split_view()`.